contentcuration/contentcuration/frontend/channelEdit/components/edit/DetailsTabView.vue

```diff
@@ -540,9 +545,21 @@
       ...mapActions('file', ['createFile', 'deleteFile']),
       update(payload) {
         this.nodeIds.forEach(id => {
+          const node = this.getContentNode(id);
+          const newNodeDetails = {
+            ...node,
+            ...payload,
+          };
+          let assessmentItems = [];
+          if (node.kind === ContentKindsNames.EXERCISE) {
+            assessmentItems = this.getAssessmentItems(id);
+          }
+          const complete = isNodeComplete({ nodeDetails: newNodeDetails, assessmentItems });
```

Comment:
Do you need to pass in files here?

Reply:
I forgot to use the logic I implemented :grin: Thanks for catching this.